modules/mbedtls/Kconfig.psa

```diff
@@ -275,9 +276,13 @@ config PSA_WANT_ALG_HKDF
 
 config PSA_WANT_ALG_PBKDF2_HMAC
 	bool
+	prompt "PSA PBKDF2_HMAC support" if !PSA_PROMPTLESS
```

Comment:
Looks like the other configs prefer use space instead of _ in the prompt text.

Reply:
Yeah, I noticed.

I felt _ was just as readable and more greppable.